crest/Assets/Crest/Crest/Scripts/Collision/QueryDisplacements.cs

```diff
@@ -21,15 +21,15 @@ public class QueryDisplacements : QueryBase, ICollProvider
 
         protected override void OnEnable()
         {
-            Debug.Assert(Instance == null);
             Instance = this;
 
             base.OnEnable();
         }
 
         protected override void OnDisable()
         {
-            Instance = null;
+            // If we set this to null, it breaks exiting play mode, as OnDisable is called but no matching call to OnEnable :/
+            //Instance = null;
```

Comment:
Should we just put a comment here saying "we don't sent Instance to null because...", and delete the commented-out line of code?

Reply:
sure